frontend/src/pages/sites/sites.tsx

```diff
@@ -1,16 +1,29 @@
-import { Button, Loader, PageLayout } from '~/libs/components/components.js';
+import React, { useCallback } from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+  Button,
+  Icon,
+  Input,
+  Loader,
+  PageLayout,
+} from '~/libs/components/components.js';
 import { AppRoute, DataStatus } from '~/libs/enums/enums.js';
-import { configureString } from '~/libs/helpers/helpers.js';
+import { configureString, initDebounce } from '~/libs/helpers/helpers.js';
 import {
-  useAppDispatch,
+  useAppForm,
   useAppSelector,
   useEffect,
   useParams,
 } from '~/libs/hooks/hooks.js';
+import { useAppDispatch } from '~/libs/hooks/use-app-dispatch/use-app-dispatch.hook.js';
```

Comment:
use proxy for imports. fix for all
```suggestion
import { useAppDispatch } from '~/libs/hooks/hooks.js';
```